pkg/karmadactl/karmadactl.go

```diff
@@ -3,6 +3,7 @@ package karmadactl
 import (
 	"flag"
 	"fmt"
+	"github.com/karmada-io/karmada/pkg/karmadactl/cmdinit"
```

Comment:
```
pkg/karmadactl/karmadactl.go:6: File is not `goimports`-ed with -local github.com/karmada-io/karmada (goimports)
	"github.com/karmada-io/karmada/pkg/karmadactl/cmdinit"

```
We usually group imports by `std packages` --> `third-part packages` --> `karmada packages`.

It's not big deal, but CI checks that...

Reply:
not fixed.

Reply:
Sorry, it has been fixed